django/db/backends/mysql/features.py

```diff
@@ -72,3 +72,13 @@ def supports_transactions(self):
         All storage engines except MyISAM support transactions.
         """
         return self._mysql_storage_engine != 'MyISAM'
+
+    @cached_property
+    def ignores_quoted_identifier_case(self):
+        """
+        See https://dev.mysql.com/doc/refman/5.7/en/identifier-case-sensitivity.html
```

Comment:
I didn't know you could skip the version number from the url's, nice.

Maybe there isn't much value, but I hope it's useful for review at least